apps/passport-client/new-components/shared/TicketCard.tsx

```diff
@@ -66,18 +68,31 @@ const DateChipContainer = styled.div`
 interface TicketCardProps {
   title: string;
   address: string;
+  ticketWidth?: number;
   ticketCount: number;
   ticketDate: string;
   cardColor: CardColor;
   imgSource?: string;
 }
 export const TicketCard = forwardRef<HTMLDivElement, TicketCardProps>(
   (
-    { imgSource, title, address, ticketCount, cardColor, ticketDate },
+    {
+      imgSource,
+      title,
+      address,
+      ticketCount,
+      cardColor,
+      ticketDate,
+      ticketWidth
+    },
     ref
   ): JSX.Element => {
     return (
-      <TicketCardContainer ref={ref} $borderColor={CARD_COLORS[cardColor]}>
+      <TicketCardContainer
+        $width={ticketWidth || 300}
```

Comment:
why do we need || 300?

Reply:
just so the screen of the components will look ok, we can remove later